SpatialGDK/Source/SpatialGDKFunctionalTests/Private/TestMaps/EventTracingMaps/EventTracingCustomGDKFilterMap.cpp

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "TestMaps/EventTracingMaps/EventTracingCustomGDKFilterMap.h"
+
+#include "EngineClasses/SpatialWorldSettings.h"
+#include "SpatialGDKFunctionalTests/SpatialGDK/EventTracingTests/EventTracingTestGameMode.h"
+#include "SpatialGDKFunctionalTests/SpatialGDK/EventTracingTests/SettingsTests/CustomGDKFilterTest.h"
+
+UEventTracingCustomGDKFilterMap::UEventTracingCustomGDKFilterMap()
+	: UGeneratedTestMap(EMapCategory::CI_NIGHTLY_SPATIAL_ONLY, TEXT("EventTracingCustomGDKFilterMap"))
+{
+	SetCustomConfig(TEXT("[/Script/SpatialGDK.SpatialGDKSettings]") LINE_TERMINATOR TEXT("bEventTracingEnabled=True")
```

Comment:
I like to just turn off clang formatting around these, and arrange them so new lines are on new lines, but minor nit